nautobot/dcim/factory.py

```diff
@@ -248,6 +266,17 @@ class Meta:
     has_comments = NautobotBoolIterator()
     comments = factory.Maybe("has_comments", factory.Faker("paragraph"), "")
 
+    has_software_images = NautobotBoolIterator()
+
+    @factory.post_generation
+    def software_images(self, create, extracted, **kwargs):
+        if not create or not DeviceTypeFactory.has_software_images.evaluate(None, None, None):
```

Comment:
That's how factoryboy grabs the next value from a factory `Declaration`. `None, None, None` is required because evaluate needs 3 positional args but they're not doing anything in this context so I just used None.